tests/test_base.py

```diff
@@ -564,6 +564,7 @@ def test_in_accepts_valid_urls(self, value):
             'http://.',
             'invalid',
             u'http://ＧＯＯＧＬＥ.com',  # full-width chars are disallowed
+            'javascript:alert()',  # TODO this is a valid schema. // is not a part of some URIs.
```

Comment:
1. `this` -> `"javascript:"`
2. `schema` -> `scheme`
3. `//` -> `"//"`

I'd also be fine with moving it, as a commented out param, to `test_it_accepts_valid_urls` and NOT test it here. It's kinda odd to test the behavior that we know is wrong.